Source/SpatialGDK/Public/SpatialInterop.h

```diff
@@ -253,6 +254,9 @@ class SPATIALGDK_API USpatialInterop : public UObject
 	// Outgoing RPCs (for retry logic).
 	TMap<FUntypedRequestId, TSharedPtr<FOutgoingReliableRPC>> OutgoingReliableRPCs;
 
+	// Pending ReserveEntity and CreateEntity requests.
+	TMap<FUntypedRequestId, USpatialActorChannel*> PendingActorRequests;
```

Comment:
Removal of the SDK is going to remove types like `FUntypedRequestId`. Could we just use `worker::RequestId`?

Reply:
We can, but we would need to wrap it to be able to hash it. Implement equals, etc.

Reply:
I would say just use worker::RequestId.Id which is just an uint32. The less wrapping objects the better.

Reply:
As discussed offline, we are keeping it this way because the type `FUntypedRequestId` is declared in `SpatialInterop.h` and it is used heavily in the file anyway.